aptos-move/framework/move-stdlib/sources/acl.move

```diff
@@ -43,4 +43,16 @@ module std::acl {
     public fun assert_contains(acl: &ACL, addr: address) {
         assert!(contains(acl, addr), error::invalid_argument(ENOT_CONTAIN));
     }
+
+    #[verify_only]
+    fun verify_remove(acl: ACL, addr: address): ACL {
+        remove(&mut acl, addr);
+        acl
+    }
+
+    #[verify_only]
+    fun verify_add(acl: ACL, addr: address): ACL {
+        add(&mut acl, addr);
+        acl
+    }
```

Comment:
These `#[verify_only]` functions might not be needed once #792 lands. I would suggest a review and landing of #792 first and then rebase this PR.

Reply:
Also, I am expecting `spec verify_remove {}` and `spec verify_add {}` blocks? Or we are just checking data invariants? In the latter case, we don't need these blocks.

Reply:
The PR you linked to doesn't seem right. Did you mean to link to a PR in the Move repo?

Reply:
Yes, sorry, I was meant to link the one in the Move repo

Reply:
@meng-xu-cs , it works great!